frontend/src/App.vue

```diff
@@ -229,7 +231,11 @@ export default {
         if(this.toggleSideBar){
           toDisplay = 'can-show-app';
         }else{
-          toDisplay = 'col-xl-9 col-lg-8 col-md-8 col-sm-10 cols-11 set-normal';
+          if(this.currentPath === '/blacksmith'){
+            toDisplay = 'col-xl-10 col-lg-9 col-md-9 col-sm-10 cols-11 set-normal';
+          }else{
+            toDisplay = 'col-xl-9 col-lg-8 col-md-8 col-sm-10 cols-11 set-normal';
```

Comment:
We do not want two different size sidebars. Choose one size for all. I think you want the top one as default